### PR TITLE
Fix 2 minor example issues for Dec. 2021 release.

### PR DESCRIPTION
**Pull request by wfansh**

### CODE REVIEW

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddResponsiveSearchAdWithAdCustomizer.java
Comment (wfansh) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddResponsiveSearchAdWithAdCustomizer.java:84]: this line (setting the default value) is never executed when running the example by providing only **customer_id** and **ad_group_id** parameters.

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddSmartCampaign.java
Comment (nwbirnie) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddSmartCampaign.java:197]: Could you write this as the positive case first? It's generally easier to mentally parse the positive case of things.

`locationIdParsed == locaitonId == null ? Long.parseUnsignedLong(locationId) : null;`

Comment (wfansh) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddSmartCampaign.java]: Done.

**Review by nwbirnie — APPROVED** [review]

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddResponsiveSearchAdWithAdCustomizer.java
Comment (devchas) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddResponsiveSearchAdWithAdCustomizer.java:84]: In the example you linked to, the line corresponding to line 84 is commented out. I think we should have the same behavior here to make it more obvious to users who are entering the params manually in the class file rather than using the command line.

**Comment by wfansh** [comment]
Please help merge this PR once approved. I myself don't have write access to this repo.
Thanks!

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddResponsiveSearchAdWithAdCustomizer.java
Comment (devchas) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddResponsiveSearchAdWithAdCustomizer.java:84]: Sorry, that was hidden in GitHub, so I thought you'd only deleted the lines.

**Review by devchas — APPROVED** [review]

**Pull request merged**
